src/components/layout/footer.js

```diff
@@ -474,6 +480,11 @@ const Footer = ({ type, is_ppc, is_ppc_redirect }) => {
                                             {localize('Payment agents')}
                                         </Link>
                                     </LinkWrapper>
+                                    <LinkWrapper>
+                                        <Link to={deriv_developer_url}>
```

Comment:
Hey @arome-binary, this will not work when clicking on these links from other languages as this is an external link out of the current domain. You will need to add the configuration onto `localized-link.js`

Reply:
Noted, seen it, working on it